library/src/androidTest/java/org/xmtp/android/library/TestHelpers.kt

```diff
@@ -41,6 +54,82 @@ class FakeWallet : SigningKey {
         get() = privateKey.walletAddress
 }
 
+class FakeSCWWallet : SigningKey {
+    private var web3j: Web3j = Web3j.build(HttpService("http://10.0.2.2:8545"))
+    private val credentials: Credentials =
+        Credentials.create("ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80")
```

Comment:
Would be nice to make this a named constant like `DEFAULT_ANVIL_CREDENTIALS` so others know where these random numbers came from.